sphinxcontrib/confluencebuilder/translator.py

```diff
@@ -7,6 +7,7 @@
 from docutils import nodes
 from docutils.nodes import NodeVisitor as BaseTranslator
 from os import path
+from jinja2 import BaseLoader, Environment
```

Comment:
(same idea as other comment) Switch to embedded template engine.

Reply:
Done